docs/sources/operators-guide/getting-started-helm-charts/_index.md

```diff
@@ -0,0 +1,269 @@
+---
+title: "Getting started with Grafana Mimir using the Helm chart"
+menuTitle: "Getting started using the Helm chart"
+description: "Learn how to get started with Grafana Mimir using the Helm chart."
+weight: 20
+---
+
+# Getting started with Grafana Mimir using the Helm chart
+
+<!-- What is this about? TODO -->
+<!-- TODO introduce the notation of <...> to mean that the user needs to set their own value here. -->
+
+## Before you begin
+
+The instructions that follow are common across any flavor of Kubernetes.
+
+The information that follows assumes that you are able to install Kubernetes, and configure and operate it.
+
+Hardware requirements:
+
+- For this tutorial a single Kubernetes node with a minimum of 4 cores and 16GiB RAM should be sufficient. Note that production setups start from 3 nodes for redundancy!
+
+Install the following software:
+
+- Either a [Prometheus server](https://prometheus.io/docs/prometheus/latest/installation/) or [Grafana Agent](https://grafana.com/docs/grafana-cloud/agent/#installing-the-grafana-agent).
+- Helm 3 or higher
+- Kubernetes 1.10 or higher
+- The kubectl command for your version of Kubernetes, and an understanding of what the command does.
+
+Verify that you have:
+
+- Access to the Kubernetes cluster
+- Persistent storage is enabled in the Kubernetes cluster and there is a default storage class set up
+- DNS service works in the Kubernetes cluster
+- An ingress controller is set up in the Kubernetes cluster
+
+  **Note:** Although this is not strictly necessary, if you want to access Mimir from outside of the Kubernetes cluster, you will need an ingress.
+
+## Install the Helm chart in a custom namespace
+
+Using a custom namespace solves problems later on because you do not have to overwrite the default namespace.
+
+1. [Create a unique Kubernetes namespace](https://kubernetes.io/docs/tasks/administer-cluster/namespaces/#creating-a-new-namespace):
+
+   ```console
+   kubectl create namespace <namespace>
+   ```
+
+1. Set up a Helm repository using the following commands:
+
+   ```console
+   helm repo add grafana https://grafana.github.io/helm-charts
+   helm repo update
+   ```
+
+   > **Note:** The Helm chart at [https://grafana.github.io/helm-charts](https://grafana.github.io/helm-charts) is a publication of the source code at [**grafana/mimir**](https://github.com/grafana/mimir/tree/main/operations/helm/charts/mimir-distributed).
```

Comment:
We could also mention here that bug reports, feature requests and contributions should go into the mimir repo.